client/sites-dashboard/components/sites-table-row.tsx

```diff
@@ -125,55 +124,51 @@ export default function SitesTableRow( { site }: SiteTableRowProps ) {
 	const displayStatusBadge = isComingSoon || site.is_private;
 
 	return (
-		<ClassNames>
-			{ ( { css } ) => (
-				<Row>
-					<Column>
-						<SiteListTile
-							contentClassName={ css`
-								min-width: 0;
-							` }
-							leading={
-								<ListTileLeading
-									href={ getDashboardUrl( site.slug ) }
-									title={ __( 'Visit Dashboard' ) }
-								>
-									<SiteIcon siteId={ site.ID } size={ 50 } />
-								</ListTileLeading>
-							}
-							title={
-								<ListTileTitle>
-									<SiteName href={ getDashboardUrl( site.slug ) } title={ __( 'Visit Dashboard' ) }>
-										{ site.name ? site.name : __( '(No Site Title)' ) }
-									</SiteName>
-									{ isP2Site && <SitesP2Badge>P2</SitesP2Badge> }
-								</ListTileTitle>
-							}
-							subtitle={
-								<ListTileSubtitle>
-									{ displayStatusBadge && (
-										<div style={ { marginRight: '8px' } }>
-											<SitesLaunchStatusBadge>
-												{ isComingSoon ? __( 'Coming soon' ) : __( 'Private' ) }
-											</SitesLaunchStatusBadge>
-										</div>
-									) }
-									<SiteUrl href={ site.URL } target="_blank" rel="noreferrer" title={ site.URL }>
-										{ displaySiteUrl( site.URL ) }
-									</SiteUrl>
-								</ListTileSubtitle>
-							}
-						/>
-					</Column>
-					<Column mobileHidden>{ site.plan.product_name_short }</Column>
-					<Column mobileHidden>July 16, 1969</Column>
-					<Column style={ { width: '20px' } }>
-						<EllipsisMenu>
-							<VisitDashboardItem site={ site } />
-						</EllipsisMenu>
-					</Column>
-				</Row>
-			) }
-		</ClassNames>
+		<Row>
+			<Column>
+				<SiteListTile
+					contentClassName={ css`
```

Comment:
IDE complains abour types here. If we are removing the component, should we move this too?

Reply:
I'm not getting an error here 🤔 TypeScript says that `css` (at least the once from the `@emotion/css` package) returns a string so this should be fine.

The error I'm seeing is that `@emotion/css` was added to the wrong `package.json`

Reply:
🤔 the dependency _has_ been added to the correct `package.json` 🤔 
Not sure why I'm getting at error.

Reply:
I see what happened - when I played with this code I had `css` imported from `@emotion/react` and not from `@emotion/css`. The first one returns `SerializedStyles` and the second one returns `string`.